lib/tests/message.test.ts

```diff
@@ -347,4 +355,610 @@ describe("Send message test", () => {
+  })
+
+  test("should send long messages and validate correct rendering", async () => {
+    const messages: string[] = []
+    const longMessages = [
+      "This is a long message with a lot of text to test the rendering of long messages in the chat.",
+      "Another long message that should be rendered correctly without any issues.",
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non arcu eget risus lacinia tincidunt ut non orci. Nullam scelerisque odio vel erat feugiat placerat.",
+      "A very lengthy message to check how the chat handles extremely long text messages. It should not break the layout or cause any issues.",
+    ]
+
+    const disconnect = channel.connect((message) => {
+      if (message.content.text !== undefined) {
+        messages.push(message.content.text)
+      }
+    })
+
+    for (const longMessage of longMessages) {
+      await channel.sendText(longMessage)
+      const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+      await sleep(2000)
+    }
+
+    await waitForAllMessagesToBeDelivered(messages, longMessages)
+
+    for (const longMessage of longMessages) {
+      expect(messages).toContain(longMessage)
+    }
+
+    disconnect()
+  }, 30000)
+
+  test("should fail to send an empty or whitespace-only message", async () => {
+    const messages: string[] = []
+
+    const disconnect = channel.connect((message) => {
+      if (message.content.text !== undefined) {
+        messages.push(message.content.text)
+      }
+    })
+
+    try {
+      await channel.sendText("")
+    } catch (error) {
+      expect(error).toBeDefined()
+      expect(error.message).toContain("Message text cannot be empty")
+    }
+
+    try {
+      await channel.sendText("   ")
+    } catch (error) {
+      expect(error).toBeDefined()
+      expect(error.message).toContain("Message text cannot be empty")
+    }
+
+    expect(messages.length).toBe(0)
+
+    disconnect()
+  }, 30000)
+
+  test("should send and receive messages in various languages correctly", async () => {
+    const messages: string[] = []
+    const textMessages = [
+      "Hello",
+      "This is a test message",
+      "你好", // Chinese
+      "مرحبًا", // Arabic
+      "こんにちは", // Japanese
+      "안녕하세요", // Korean
+      "Hola", // Spanish
+    ]
+
+    const disconnect = channel.connect((message) => {
+      if (message.content.text !== undefined) {
+        messages.push(message.content.text)
+      }
+    })
+
+    for (const textMessage of textMessages) {
+      await channel.sendText(textMessage)
+      const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+      await sleep(2000)
+    }
+
+    await waitForAllMessagesToBeDelivered(messages, textMessages)
+
+    for (const textMessage of textMessages) {
+      expect(messages).toContain(textMessage)
+    }
+
+    disconnect()
+  }, 30000)
+  //Task created CSK-284
+  test("should fail to edit a deleted message", async () => {
```

Comment:
this test doesn't really check anything, right?

Reply:
yeah, I think I will come back to it after we will consider message delete behaviour. Meantime cleaned up.